### PR TITLE
Orassans

### PR DESCRIPTION
**Pull request by onigamii**
## Changes
- Updated the Orassans patch and patched in the new DMR.
- Orassan Heavy Pistol ammo from 12x64 to 8x35mm.
- Orassan SMG ammo from 6x24mm to 6x18mm.
- Added custom projectile for the Cryo Launcher.

## Testing

- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested

### CODE REVIEW

File: Patches/Orassans/ThingDefs_Misc/Apparel_Hats.xml [hunk 1]
```diff
@@ -1,74 +1,83 @@
...
-			<li Class="PatchOperationAdd">
-				<xpath>Defs/ThingDef[defName = "Apparel_OrassanSacriArmorHelmet"]/statBases</xpath>
-				<value>
-					<Bulk>5</Bulk>
-					<WornBulk>1</WornBulk>
-				</value>
-			</li>
+				<!-- ========== Orassan Power Helmet ========== -->
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/ThingDef[defName="Apparel_OrassanPowerArmorHelmet"]/statBases</xpath>
```
Comment (N7Huntsman) [inline Patches/Orassans/ThingDefs_Misc/Apparel_Hats.xml:13]: Several helmets, like the S.A.C.R I and Exoskeleton helmets aren't patched.

File: Patches/Orassans/ThingDefs_Misc/Apparel_Various.xml [hunk 2]
```diff
@@ -1,133 +1,153 @@
...
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Orassans</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+			
+				<!-- ========== Orassan Power Armor ========== -->
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/ThingDef[defName="Apparel_KittyArmor"]/statBases</xpath>
```
Comment (N7Huntsman) [inline Patches/Orassans/ThingDefs_Misc/Apparel_Various.xml:13]: The exoskeletal armor also needs patched.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
The cat's eye juice bottles needs its tools patched in line with other alcohols. The cryo launcher, missile launcher, and shotgun also needs a melee patch.

**Comment by Tostov** [comment]
iirc, the mod also adds like, CatLavenderCloth or something, which is a devilstrand equivalent in vanilla and probably needs its stats reduced/fixed for CE levels and to be added to the soft armor catergory.

Also possibly the race's leathers.

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**